packages/mobile/src/sentry/Sentry.ts

```diff
@@ -1,32 +1,57 @@
 import * as Sentry from '@sentry/react-native'
 import DeviceInfo from 'react-native-device-info'
 import { select } from 'redux-saga/effects'
-import { SENTRY_CLIENT_URL } from 'src/config'
+import { sentryTracesSampleRateSelector } from 'src/app/selectors'
+import { DEFAULT_FORNO_URL, SENTRY_CLIENT_URL } from 'src/config'
+import networkConfig from 'src/geth/networkConfig'
 import Logger from 'src/utils/Logger'
 import { currentAccountSelector } from 'src/web3/selectors'
 
 const TAG = 'sentry/Sentry'
 
 // Set this to true, if you want to test Sentry on dev builds
+// Set tracesSampleRate: 1 to capture all events for testing performance metrics in Sentry
 export const SENTRY_ENABLED = !__DEV__ || false
+export const sentryRoutingInstrumentation = new Sentry.ReactNavigationInstrumentation()
 
-if (SENTRY_ENABLED) {
-  installSentry()
-} else {
-  Logger.info(TAG, 'Sentry not enabled')
-}
+export function* initializeSentry() {
+  if (!SENTRY_ENABLED) {
+    Logger.info(TAG, 'Sentry not enabled')
+    return
+  }
 
-// This should be called as early in the lifecycle of the app as possible.
-function installSentry() {
   if (!SENTRY_CLIENT_URL) {
     Logger.info(TAG, 'installSentry', 'Sentry URL not found, skiping instalation')
     return
   }
+
+  const tracesSampleRate = yield select(sentryTracesSampleRateSelector)
+  const tracingOrigins = [
+    DEFAULT_FORNO_URL,
+    networkConfig.blockchainApiUrl,
+    networkConfig.odisUrl,
+    networkConfig.komenciUrl,
+    networkConfig.providerFetchUrl,
+    networkConfig.simplexApiUrl,
+    networkConfig.fetchUserLocationDataUrl,
+    networkConfig.komenciLoadCheckEndpoint,
+    networkConfig.walletConnectEndpoint,
+    networkConfig.inhouseLiquditiyUrl,
```

Comment:
Some of these are cloud functions sharing the same domain, can we track the base domain instead?

Reply:
I think we should also add Komenci's URL? If we have it..

Reply:
i had this but removed because i couldn't verify that the traces for it are coming through...in this PR we are only tracking the navigation traces and if i understand correctly, the http calls to komenci do not impact the navigation? 

do you think we should include all endpoints in this tracingOrigins array? not sure if you saw my comment[https://github.com/valora-inc/wallet/pull/1632#discussion_r772528463] but that is how i ended up with this current list, let me know if you think we should go back to including everything?

Reply:
I think this is fine for now, we can look more into it when/if we add specific performance monitoring to the verification flow